db/schema.rb

```diff
@@ -259,6 +257,7 @@
     t.string   "sold_out_dropdown_message", limit: 255
     t.string   "sold_out_customer_info",    limit: 255
     t.integer  "season",                                default: 2021,           null: false
+    t.string   "reminder_type",                         default: "Never",        null: false
```

Comment:
This seems to be the only change from the db migration.

I would suggest doing a manual change to the db schema.rb file to include this line so that other tables continue using btrees and stays consistent with the [golden repo's schema](https://github.com/armandofox/audience1st/blob/main/db/schema.rb).

Reply:
**NO!!!!!!!!!!!!!!**

YOU SHOULD NEVER, EVER, EVER manually change schema.rb.  it is managed by the migration subsystem for that very reason.